<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministorPage</class>
 <widget class="QWidget" name="AdministorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/backgroud.png);</string>
  </property>
  <widget class="QLineEdit" name="in_adm">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>295</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="adm_info">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;输入管理密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adm_sure">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>84</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
